src/main/java/dto/HistoriesDto.java

```diff
@@ -0,0 +1,24 @@
+package dto;
+
+import domain.AbsenceLevel;
+import domain.AttendanceHistory;
+import domain.AttendanceResult;
+import java.util.List;
+import java.util.Map;
+
+public record HistoriesDto(
+        String username,
+        List<HistoryDto> histories,
+        Map<AttendanceResult, Integer> attendanceAllResult,
+        String classifyAbsenceLevel
+) {
+
+    public static HistoriesDto of(String username, List<AttendanceHistory> histories,
+                                  Map<AttendanceResult, Integer> attendanceAllResult,
+                                  AbsenceLevel classifyAbsenceLevel) {
+        List<HistoryDto> historyDtoList = histories.stream().map(history -> {
+            return new HistoryDto(history.getAttendanceTime(), history.getAttendanceResult().getResult());
+        }).toList();
```

Comment:
메서드 체이닝을 사용할 떄는 메서드마다 개행을 해주면 가독성이 좋아진다고 생각해요~
( 람다에서 단일 실행은 return 문을 생략해도 괜찮습니다 😉 )
```suggestion
        List<HistoryDto> historyDtoList = histories.stream()
                .map(history -> new HistoryDto(history.getAttendanceTime(), history.getAttendanceResult().getResult()))
                .toList();
```

Reply:
return 생략 및 개행 처리 했습니다!